hyperkube/10-containernet.conf

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "containernet",
+  "type": "flannel",
+  "subnetFile": "/var/run/flannel/subnet.env",
```

Comment:
Ah, now I understand. you switch the directories with the mount flag. See my comment in the docker run command. IMHO, the file should stay in /run because this directory is non-persistent, while /var is persistent. Any reason to change the location to non-default place?